src/components/presentation/shareButton/Share.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { render, cleanup } from 'react-testing-library';
+import Share from './Share';
+
+const props = {
+  articleSlug: 'ogbuke-kosilowo-345542',
+  articleTitle: 'Kiging to the core',
+};
+
+describe('Share component test', () => {
+  afterEach(cleanup);
```

Comment:
Good job with the cleanup. However, we're already handling for this in our `setupTest.js` file

Reply:
Please remove it.

Reply:
Done!